README.md

```diff
@@ -15,12 +15,16 @@ This is a Carbon Five-flavored convenience skeleton project for React with TypeS
 To get started, make sure you have Node 8.10+ and Yarn installed, and then generate your project like this:
 
 ```
-$ npx spraygun -t react-ts <project-directory>
+$ npx spraygun -t react <project-directory>
 ```
 
 If you'd like to use spraygun-react-ts with a Rails API backend, follow this guide:
 
-> [Using spraygun-react with a Rails backend](https://github.com/carbonfive/spraygun-react/blob/master/docs/how-to-use-with-rails-backend.md)
+> [Using spraygun-react-ts with a Rails backend](./docs/how-to-use-with-rails-backend.md)
+
+If you'd like to use spraygun-react-ts with Redux, here are potential file structures:
+
+> [Redux file structures](./docs/example-redux-file-structure.md)
```

Comment:
I think these docs will probably be the same for JS and TS. What do you think about linking to the spraygun-react versions of these 2 docs so that we don't have to maintain separate copies in this repo?

Reply:
ooh good call!